### DISCUSSION TITLE
binding problem on custom control

### DISCUSSION THREAD

**Discussion by movinator** [dotnet/maui#9412] [General]
Hi,

I'm working on a custom control (according manual of ContentView). The trivial part worked fine.
Now I added some bindable Properties, but I get compile errors, telling me, that "expression does not have a name".
Expression is the nameof(...) at BindableProperty.Create(...).

I checked manual and my sources multiple times - can't find a buggy difference.

How can I narrow that error or find out, what did I wrong?

**Comment by xtuzy** [comment]
please show your custom control code

**Comment by movinator** [comment]
Thank you very much for your attention.

I started stupidly copy/pasting code from manual.
``public static readonly BindableProperty CardTitleProperty = BindableProperty.Create(nameof(CardTitle), typeof(string), typeof(CardView), string.Empty);
``
The part "nameof(CardTitle)" generated an error.

So I changed original code to
``public static readonly BindableProperty CardTitleProperty = BindableProperty.Create("CardTitle", typeof(string), typeof(CardView), string.Empty);
``
which seems to work.
